test/model_simulation/simulate_ODEs.jl

```diff
@@ -99,64 +116,88 @@ let
         du[2] = k2 * log(12 + X) * X - k3 * log(3 + Y) * Y
         du[3] = k3 * log(3 + Y) * Y - log(5, 6 + k4) * Z
     end
-    push!(identical_networks_1, reaction_networks_weird[2] => real_functions_5)
+    push!(catalyst_networks, reaction_networks_weird[2])
+    push!(manual_networks, real_functions_5)
+    push!(u0_syms, [:X, :Y, :Z])
+    push!(ps_syms, [:k1, :k2, :k3, :k4])
 
-    for (i, networks) in enumerate(identical_networks_1)
+    for (rn_catalyst, rn_manual, u0_sym, ps_sym) in zip(catalyst_networks, manual_networks, u0_syms, ps_syms)
         for factor in [1e-2, 1e-1, 1e0, 1e1]
```

Comment:
I considered changing some of these when we re-run tests (and I did some stochastic ones). But practically, we are not talking about any really time, so I left them as they were.